src/FileCache.php

```diff
@@ -150,7 +150,12 @@ public function set(string $key, mixed $value, null|int|DateInterval $ttl = null
             }
         }
 
-        $result = @touch($file, $expiration);
+        $result = false;
+
+        if (@touch($file, $expiration)) {
+            clearstatcache(false, $file);
```

Comment:
```suggestion
            clearstatcache(true, $file);
```

It works only when clear_realpath_cache is true.

Reply:
And we don't care about realpath cache, so passing any parameters does not make sense.

Reply:
I'm tested this case. `clearstatcache(false, $file);` is cleared cache of modified time.